index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Awesome Books in ES6</title>
+  </head>
+  <body>
+    <header>
+      <h1>Awesome Books in ES6</h1>
+      <nav>
+        <ul id="menu">
+          <li><a id="listMenuLink" href="#">List</a></li>
+          <li><a id="addMenuLink" href="#">Add new</a></li>
```

Comment:
- [ ] Your app is unresponsive, and hence not working as intended. When the _Add new_ link is clicked nothing happens. Kindly make sure that it is possible to **add and remove books** from the list.

![image](https://user-images.githubusercontent.com/12745474/153852696-428ef542-e0cf-44f3-b5de-2f59e1cbd5d2.png)

Reply:
@rloterh this is because it seems github pages doesn't run `npm install`. This is why we don't get luxon installed and therefor there is an error on JS and of course nothing works.

If we need to install luxon from npm, we need to have a build script. we cannot make gh pages and expect things to work. Isn't it? Am I missing some point?

Reply:
@ahangarha 
I actually cloned and run your project locally, and the results were the same as your GitHub version. The issue is not with npm. Kindly ensure to go through your code thoroughly.